app/src/main/java/com/portfolio/tasky/ToolbarHandlerImpl.kt

```diff
@@ -0,0 +1,23 @@
+package com.portfolio.tasky
+
+import android.view.animation.TranslateAnimation
+import androidx.appcompat.widget.Toolbar
+import com.portfolio.tasky.globals.Constants
+
+
+class ToolbarHandlerImpl : ToolbarHandler {
```

Comment:
I like that you did this

But I got a question

In which part of the Tasky app will you use a Toolbar?

Reply:
In Login, Registration, Task Details, Event Detail and Reminder Detail screens. 

Since there are dates to be shown in the Tasks, Events and Reminder screens, I'll be able to utilise the current implementation for setting the text and animation.